dsgrid/config/dataset_config.py

```diff
@@ -96,19 +96,30 @@ class DataSchemaType(DSGEnum):
         Applies to datasets for which the data are provided in full.
         """,
     )
+    ONE_TABLE = EnumValue(
+        value="one_table",
+        description="""
+        One_table data schema with load_data table. 
+        Typically appropriate for small, low-temporal resolution datasets.
+        """,
+    )
 
 
 class DSGDatasetParquetType(DSGEnum):
     """Dataset parquet types."""
 
     LOAD_DATA = EnumValue(
         value="load_data",
-        description="Load data file with id, timestamp, and load value columns",
+        description="""
+        In STANDARD data_schema_type, load_data is a file with ID, timestamp, and load value columns. 
+        In ONE_TABLE data_schema_type, load_data is a file with multiple data dimension, timestamp, and load value columns.
```

Comment:
"multiple data dimension and load value columns"? "timestamp" is just another data dimension after all ...

Reply:
Shoot, and for all of these, instead of "load value" it should say "metric value"

Reply:
This is also making me question our rampant use of "load_data", but I think that's okay--even though we want to register multiple metrics, we are focused on load-relevant data.

Reply:
As of now the standard data schema requires a timestamp column. This was a simplification we made since we expect most datasets to follow this convention. If we will support a dataset with a different layout then we would need to make a new schema type - that could be completely generic or specific.

We could change the standard schema now at a cost of a few development days.